app/src/main/res/layout/onboarding_header.xml

```diff
@@ -2,19 +2,34 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<FrameLayout
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:tools="http://schemas.android.com/tools"
-        android:id="@+id/onboarding_header"
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/conclusion"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="-4dp"
```

Comment:
Where is this marginTop -4dp coming?

Reply:
I understand this is to counteract the wordmark 32dp margin. However, the margin between the header and Firefox Browser logo still looks much bigger than 28dp that is seen in the design. Let's have a check with UX since the designs doesn't have a proper whole number (27.77dp). This might not be something they intended to change.